<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONF_ActionSlicePointCloudOptions</class>
 <widget class="QWidget" name="ONF_ActionSlicePointCloudOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wd_toolbar" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lb_thickness">
           <property name="text">
            <string>Epaisseur des couches :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_thickness">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.998999999999796</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_spacing">
           <property name="text">
            <string>Espace entre les couches :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_space">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Incr√©ment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_1mm">
           <property name="text">
            <string>1 mm</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_1cm">
           <property name="text">
            <string>1 cm</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_10cm">
           <property name="text">
            <string>10 cm</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_1m">
           <property name="text">
            <string>1 m</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_singleStep"/>
 </buttongroups>
</ui>
